core/src/main/scala/cromwell/core/package.scala

```diff
@@ -5,9 +5,12 @@ import wdl4s.values.{SymbolHash, WdlValue}
 import scalaz._
 
 package object core {
+  class WorkflowContext(val root: String)
+  class CallContext(override val root: String, val stdout: String, val stderr: String) extends WorkflowContext(root)
   type ErrorOr[+A] = ValidationNel[String, A]
 
   type LocallyQualifiedName = String
   case class CallOutput(wdlValue: WdlValue, hash: Option[SymbolHash])
   type CallOutputs = Map[LocallyQualifiedName, CallOutput]
+  type EvaluatedRuntimeAttributes = Map[String, WdlValue]
```

Comment:
I'd also prefer that cases where one thinks a type is warranted that a real type (i.e. case class) is used instead of a type alias